docs/source/tuning_strategies.md

```diff
@@ -76,34 +76,34 @@ User can set the accuracy criteria by specifying the `higher_is_better`, `criter
 ``` python
 from neural_compressor.config import AccuracyCriterion
 
-accuracy_criterion=AccuracyCriterion(
-    higher_is_better=True, # optional. 
-    criterion='relative', # optional. Available values are 'relative' and 'absolute'.
-    tolerable_loss=0.01, # optional.
+accuracy_criterion = AccuracyCriterion(
+    higher_is_better=True,  # optional. 
+    criterion='relative',  # optional. Available values are 'relative' and 'absolute'.
+    tolerable_loss=0.01,  # optional.
 )
 ```
 
 ### Traverse 
-Once the `tuning space` was constructed, user can specify the traverse logic by setting the `quant_level` field with `0` or `1` in the `PostTrainingQuantConfig` (`QuantizationAwareTrainingConfig`), or the `strategy` field with strategy name in the `TuningCriterion`. The priority of `quant_level` is higher than `strategy`, which means the `quant_level` should be set to `1` if user wants to specify the traverse logic by strategy name. The design and usage of each traverse logic is introduced in the following session.
+Once the `tuning space` was constructed, user can specify the traverse logic by setting the `quant_level` field with `0` or `1` in the `PostTrainingQuantConfig`, or the `strategy` field with strategy name in the `TuningCriterion`. The priority of `quant_level` is higher than `strategy`, which means the `quant_level` should be set to `1` if user wants to specify the traverse logic by strategy name. The design and usage of each traverse logic is introduced in the following session.
 
 ## Traverse Logic
 
 ### O0 
```

Comment:
this section title is incorrect. "O0" there is no any word to map in the context.

and pls explain what's the possible values for this parameter, what's the difference between "quant_level being 0" and "quant_level being 1".